src/display.hpp

```diff
@@ -43,7 +42,6 @@ class team;
 struct overlay;
 
 namespace halo {
-	class manager;
 }
```

Comment:
There's a few places like this, where the `namespace x {` and `}` lines could be removed too.